### PR TITLE
Past Editions

### PR DESCRIPTION
**Pull request by arutr**
First version of past editions page - enough to satisfy the project leader, I hope... ;)

The contact form mechanism has been commented out, and replaced with a simple `mailto:` anchor, as the e-mail notification system isn't configured? One enquiry has been left untouched since January!

### CODE REVIEW

File: client/layout/footer/footer.pug [hunk 1]
```diff
@@ -24,13 +24,16 @@ mixin footer
...
           a(href=contentControl.bylawLink) By-law for Poland 2.0
-          +modalOpenButton("contact-form-modal", "/contact")(title="Open contact form") Contact
+          //+modalOpenButton("contact-form-modal", "/contact")(title="Open contact form") Contact
+          a(href="mailto:contact@poland20.com") Contact
         .site-footer__copyright
           small © #{moment().year()} Poland 2.0
         address.site-footer__credits
           small Created by <a href="https://github.com/jakow" target="_blank" rel="noopener">Jakub Kowalczyk</a>
+          <br/>
+          small Maintained by <a href="mailto:artur@komoter.pl">Artur Komoter</a>
```
Comment (jakow) [inline client/layout/footer/footer.pug:34]: ;D

File: @config/client.config.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ const cssLoaders = [
 ];
 
 // root build directory for the SERVER. Client is in /client/
-const buildDir = path.resolve(__dirname, '..', 'build');
+const buildDir = path.resolve(__dirname, '..', isProduction ? 'build' : 'static');
```
Comment (jakow) [inline @config/client.config.js:29]: What is the reason that we change where server files are placed? The buildDirs is where TS -> JS conversion sits, static files should be in a different place.

**Review by jakow — APPROVED** [review]
LGTM, but if you could clarify that one comment 🥇

**Comment by jakow** [comment]
Hoping this works

File: @config/client.config.js (hunk 2 above)
Comment (arutr) [reply @config/client.config.js:29]: Without this generated client scripts are inaccessible from `/build/client/` without server aliasing for `/static/client`.

It's only necessary for development purposes. Shouldn't affect production ;)

**Pull request merged**